src/Loggers/LogGraph.jl

```diff
@@ -0,0 +1,84 @@
+"""
+	log_graph
+"""
+function log_graph(logger::TBLogger, g::AbstractGraph; step = nothing, nodelabel::Vector{String} = map(string, vertices(g)), nodeop::Vector{String} = map(string, vertices(g)), nodedevice::Vector{String} = fill("cpu", nv(g)), nodevalue::Vector{Any} = fill(nothing, nv(g)))
+	nv(g) == length(nodelabel)  || throw(ArgumentError("length of nodelable must be same as number of vertices"))
+	nv(g) == length(nodeop)  || throw(ArgumentError("length of nodeop must be same as number of vertices"))
+	nv(g) == length(nodedevice)  || throw(ArgumentError("length of nodedevice must be same as number of vertices"))
+	nv(g) == length(nodevalue)  || throw(ArgumentError("length of nodevalue must be same as number of vertices"))
+	summ = SummaryCollection(graph_summary(g, nodelabel, nodeop, nodedevice, nodevalue))
+    write_event(logger.file, make_event(logger, summ, step=step))
+end
+
+function graph_summary(g, nodelabel, nodeop, nodedevice, nodevalue)
+	nodes = Vector{NodeDef}()
+	for v in vertices(g)
+		name = nodelabel[v]
+		op = nodeop[v]
+		input = [nodelabel[x] for x in inneighbors(g, v)]
+		device = nodedevice[v]
+		attr = Dict{String, AttrValue}()
+		x = nodevalue[v]
+		if isa(x, AbstractString)
+			attr["value"] = AttrValue(s = Vector{UInt8}(x))
+			attr["dtype"] = AttrValue(_type = jltype2tf(typeof(x)))
+		elseif isa(x, Integer)
+			attr["value"] = AttrValue(i = Int64(x))
+			attr["dtype"] = AttrValue(_type = jltype2tf(typeof(x)))
+		elseif isa(x, Real)
+			attr["value"] = AttrValue(f = Float32(x))
+			attr["dtype"] = AttrValue(_type = jltype2tf(typeof(x)))
+		elseif isa(x, Bool)
+			attr["value"] = AttrValue(b = x)
+			attr["dtype"] = AttrValue(_type = jltype2tf(typeof(x)))
+		elseif isa(x, AbstractArray)
+			shape = TensorShapeProto(dim = [TensorShapeProto_Dim(size = d) for d in (collect(size(x)))])
+			t = TensorProto(dtype = jltype2tf(eltype(x)), tensor_shape = shape, tensor_content = serialize_proto(string(x)))
+			attr["value"] = AttrValue(tensor = t)
+			attr["_output_shapes"] = AttrValue(list = AttrValue_ListValue(shape = [shape]))
+		elseif isa(x, Tuple)
+			attr["value"] = AttrValue(list = AttrValue_ListValue(s = [Vector{UInt8}(repr(y)) for y in x]))
+			shape = TensorShapeProto(dim = [TensorShapeProto_Dim(size = length(x))])
+			attr["_output_shapes"] = AttrValue(list = AttrValue_ListValue(shape = [shape]))
+		elseif isa(x, Function)
+			attr["value"] = AttrValue(func = NameAttrList(name = repr(x)))
+		elseif x == nothing
+			#donothing
+		else
+			throw("unhandled nodevalue type $(typeof(x))")
```

Comment:
I think this case shouldn't throw an error. I'll just `#do nothing` in this case.